libs/http2-manager/src/HTTP2/Client/Manager.hs

```diff
@@ -0,0 +1,11 @@
+module HTTP2.Client.Manager
+  ( HTTP2Manager,
+    TLSEnabled,
+    defaultHTTP2Manager,
+    http2ManagerWithSSLCtx,
+    withHTTP2Request,
+    ConnectionAlreadyClosed (..),
+  )
```

Comment:
also export a way to inspect and change the cache limit here?  (or at least initialize.)